configure

```diff
@@ -7,7 +7,7 @@
 
 import sys
 if not sys.version_info >= (3, 4):
-    print("openage requires Python 3.4 or higher")
+    print("Openage requires Python 3.4 or higher.")
```

Comment:
heh :) `openage` is actually meant to be a lowercase-word, so I'm not sure if we should capitalize it even on sentence beginnings.